assets/components/pages/project/curate/VariantData.js

```diff
@@ -147,10 +152,240 @@ class VariantData extends React.Component {
     showAll: false,
   };
 
+  getGQFrequencies() {
```

Comment:
These methods are very similar, would you consider creating a method like `getPreparedBinnedData(data, ...other config options)`